x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlResponseListener.java

```diff
@@ -143,12 +143,16 @@ public ActionListener<EsqlQueryResponse> wrapWithLogging() {
         return ActionListener.wrap(r -> {
             onResponse(r);
             // At this point, the StopWatch should already have been stopped, so we log a consistent time.
-            LOGGER.info("Successfully executed ESQL query in [{}]ms: [{}]", stopWatch.stop().getMillis(), esqlQuery);
+            LOGGER.info(
+                "Finished execution of ESQL query.\nQuery string: [{}]\nExecution time: [{}]ms",
```

Comment:
@alex-spies I've noticed these messages today while testing stuff and I was wondering why they look like `Failed executing ESQL query in [8]ms: [show functions | where variadic is true | keep synopsis]`. I understand the square brackets surrounding the query itself (to make it stand out and be clearly readable), but I don't get why we have `[8]ms`? I don't think they are necessary, are they?

Reply:
Yeah, `[8]ms` looks a bit irritating to myself as well. However, this is consistent with how our [contribution instructions ](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#logging)say that we should log, there's even exactly this example where they also write `... failed in [{}]ms`.

Reply:
Imho that's a mistake. I searched ES code for both `[{}]ms` and `[{}ms]` and the second approach is much more common.